scripts/bundle-and-gitignore-deps.js

```diff
@@ -10,7 +10,10 @@ const shouldIgnore = []
 
 arb.loadVirtual().then(tree => {
   for (const node of tree.children.values()) {
-    if (node.dev || node.isLink) {
+    const has = (obj, key) => Object.prototype.hasOwnProperty.call(obj, key)
+    const nonProdWorkspace =
+      node.isWorkspace && !(has(tree.package.dependencies, node.name))
```

Comment:
@isaacs this makes me think we should have a better API in Arborist to tell the difference between a prod/dev/peer declared workspace vs a workspace-declared-only 🤔